.circleci/config.yml

```diff
@@ -518,18 +518,18 @@ workflows:
             - check-mattermost-vet
             - check-golangci-lint
             - build-api-spec
-      - upload-s3-sha:
-          context: mattermost-ci-pr-builds-s3
-          requires:
-            - build
-      - upload-s3:
-          context: mattermost-ci-pr-builds-s3
-          requires:
-            - build
-      - build-docker:
-          context: matterbuild-docker
-          requires:
-            - upload-s3-sha
+#      - upload-s3-sha:
```

Comment:
was this intentional?

Reply:
I think this came because I didn't pointed this to `cloud` branch when creating the PR. It should be ok now.